gsad/src/gsad_http.h

```diff
@@ -185,7 +185,7 @@ int send_response (http_connection_t *connection, const char *content,
                    int status_code, const gchar *sid,
                    content_type_t content_type,
                    const char *content_disposition,
-                   size_t content_length);
+                   size_t content_length, int allow_caching);
```

Comment:
Please don't add a param for caching to all functions. It make the functions signature only longer and calls more and more unreadable.